torch/_dynamo/variables/torch.py

```diff
@@ -608,11 +608,27 @@ def fn_with_prim_types(x):
                 and self.value in tensortype_to_dtype
                 and len(args) == 1
                 and isinstance(args[0], ListVariable)
-                and args[0].is_python_constant()
             ):
+                all_ndarray = True
                 for x in args[0].items:
-                    if isinstance(x.value, np.generic):
+                    if not isinstance(x, variables.NumpyNdarrayVariable):
+                        all_ndarray = False
+                    if x.is_python_constant() and isinstance(
```

Comment:
Are np.generics, when wrapped as NumpyNdarrayVariables, still tracked as python constants? Are they not tracked as 0-D arrays?

Reply:
Removed, the reason for disabled test is that not tracking as ndarray no longer exists

Reply:
Actually, I think we should technically allow lists of ints or symints too... but let's leave that for another time.